### DISCUSSION TITLE
Troubleshooting install, file not found for xml files in data folder

### DISCUSSION THREAD

**Discussion by cmkyee** [mutationpp/Mutationpp#207] [Q&A] [answered]
Hey all, brand new to M++, coming over from ACE. 

I'm on MacOS 12.3.1 and using CMake 3.23.1, MacPorts 2.7.2, and Xcode 13.3.1 for command line. I'm assuming this is just a user error, but I'm pretty stuck. I followed the install guide and it looks like MPP has installed correctly, but I always receive file not found errors when any of the functions try to call up anything in the data folder. I assume it's something that I did wrong with my .bash_profile.

My file structure is:
/User/name/Mutationpp
/User/name/Mutationpp/data
/User/name/Mutationpp/build/install (with bin and lib in the install folder)

My .bash_profile is in /User/name and I'm using nano to edit it.

```
export MPP_DIRECTORY=/Users/colinyee/Mutationpp
export MPP_DATA_DIRECTORY=$MPP_DIRECTORY/data
export PATH=$MPP_DIRECTORY/build/install/bin:$PATH
export DYLD_LIBRARY_PATH=$MPP_DIRECTORY/build/install/lib:$DYLD_LIBRARY_PATH
```

Should my .bash_profile be named something else? Or should I change the folder structure somehow? Thanks in advance.

**Comment by mgoodson-cvd** [comment]
I would usually put this in your `.bashrc`, but on Mac this may not matter. What do you get if you call `echo $MPP_DATA_DIRECTORY` and `ls $MPP_DATA_DIRECTORY`? That will tell you if you've set things up correctly.

**Reply by cmkyee** [reply]
I don't get anything from either, so looks like it's not set up correctly. I made `.bashrc` and copied the exact same contents into it and restarted but still have the same issue.

```
colinyee@Colins-MBP ~ % echo $MPP_DATA_DIRECTORY

colinyee@Colins-MBP ~ % ls $MPP_DATA_DIRECTORY
Applications		Library			Pictures
Desktop			Movies			Public
Documents		Music			training-materials
Downloads		Mutationpp

```

**Reply by mgoodson-cvd** [reply]
Most likely you have `MPP_DIRECTORY` incorrect, maybe the `Users` should be lowercase? Not sure. Just run a `pwd` from your Mutationpp directory and check. Based on what you show above, it looks like Mutationpp is in your home directory? If so, then it may be easiest to just use `export MPP_DIRECTORY=~/Mutationpp`.

**Comment by cmkyee** [comment, marked as answer]
Ok, so I figured out the problem. Instead of putting the `export` lines in `.bashrc` or `.bash_profile` , I needed to put them in `.zprofile` in the home directory. Now `echo $MPP_DATA_DIRECTORY` is correctly returning the location and `checkmix` is working with all of the mixture profiles.


```
colinyee@Colins-MBP ~ % echo $MPP_DATA_DIRECTORY
/Users/colinyee/Mutationpp/data
```

**Reply by mgoodson-cvd** [reply]
That would make sense if you're using `zsh` instead of `bash`. You can `echo $SHELL` to confirm. Glad you got it resolved!

**Reply by cmkyee** [reply]
```
colinyee@Colins-MBP ~ % echo $SHELL
/bin/zsh
```

Yeah, thanks!
